From 15f51edf989cc47588c3d8635a62c255cdeb0072 Mon Sep 17 00:00:00 2001
From: Daniel Cheng <dcheng@chromium.org>
Date: Wed, 6 Oct 2021 03:23:23 -0700
Subject: [PATCH] Turn on hardened mode

Enable various hardening checks in absl which would otherwise be UB.
---
 third_party/abseil-cpp/absl/base/options.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/third_party/abseil-cpp/absl/base/options.h b/third_party/abseil-cpp/absl/base/options.h
index 71bafb39f2124..212891c5067f1 100644
--- a/third_party/abseil-cpp/absl/base/options.h
+++ b/third_party/abseil-cpp/absl/base/options.h
@@ -179,6 +179,6 @@
 // checks enabled by this option may abort the program in a different way and
 // log additional information when `NDEBUG` is not defined.

-#define ABSL_OPTION_HARDENED 0
+#define ABSL_OPTION_HARDENED 1

 #endif  // ABSL_BASE_OPTIONS_H_
--
2.49.0.rc0.332.g42c0ae87b1-goog
